iris/tests/download_manager/open_download_folder.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from iris.api.helpers.download_manager_utils import download_file, DownloadFiles, downloads_cleanup
+from iris.test_case import *
+
+
+class Test(BaseTest):
+
+    def __init__(self):
+        BaseTest.__init__(self)
+        self.meta = 'Open download folder.'
```

Comment:
Please update self.meta to match testcase's title from testrail("Open Download Folder" opens the folder which Firefox is set to save its downloads.)